jupyterlite_sphinx/_try_examples.py

```diff
@@ -407,7 +409,10 @@ def insert_try_examples_directive(lines, **options):
         + [""]
         + ["    " + line for line in lines[left_index:right_index]]
         + [""]
-        + lines[right_index:]
     )
 
+    # Append the remainder of the docstring, if there is any
+    if right_index < len(lines):
+        new_lines += lines[right_index:]
```

Comment:
should this also include the empty line you're appending, i.e.:
```suggestion
    # Append the remainder of the docstring, if there is any
    if right_index < len(lines):
        new_lines += [""] + lines[right_index:]
```
(and remove the one above, at the end of the construction of `new_lines`)

Reply:
Ah, makes sense to me!